docs/content-guidelines.md

```diff
@@ -0,0 +1,198 @@
+# Content Guidelines
+
+This page contains instructions on how to create documentation for Luigi. It defines the rules for each of these topics:
+* [Structure](#structure)
+* [Metadata](#metadata)
+* [Audience and language](#audience-and-language)
+* [Format](#format)
+* [Links](#links)
+* [Screenshots and diagrams](#screenshots-and-diagrams)
+* [Glossary](#glossary)
+
+## Overview
+
+Luigi documentation exists in the form of Markdown files on GitHub, which are then transferred to the main documentation page using Sapper. Find more about what GitHub-flavored Markdown is [here](https://github.github.com/gfm).
+
+Documentation is located in the `luigi/docs` folder in the [Luigi repository](https://github.com/SAP/luigi).
```

Comment:
```suggestion
Documentation is located in the `luigi/docs` folder in the [Luigi repository](https://github.com/SAP/luigi).
```
it can also be `documentation resides` if you want to keep the active voice. But `is located` is also good :)